app/policies/auth.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Auth
+  def self.authorized?(user, policy, method)
+    Pundit.policy!(user, policy).send(method)
+  end
+end
```

Comment:
Would it be better to put this in a module / mixin and mix it in on user.rb?

`User.authorized?(self, policy)`

reads a little bit better.

Reply:
Only thing mixing it back in to user is the methods are right back where they started 😃 . Although  obviously the user is more maintainable if they're in a module. I like a distinct class as it has clear inputs and outputs with no side effects. Send in a user, service, and action and it gives you a response. Also easier to turn into a service later on if need be.